src/java_bytecode/java_types.cpp

```diff
@@ -389,6 +414,55 @@ typet java_type_from_string(
 
       return java_reference_type(symbol_type);
     }
+  case '*':
+  {
+#ifdef DEBUG
+    std::cout << class_name << std::endl;
+#endif
+    // parse name of type variable
+    INVARIANT(src[src.size()-1]==';', "Generic type name must end on ';'.");
+    PRECONDITION(!class_name.empty());
+//    INVARIANT(
+//      src.substr(1, src.size()-2)=="*",
+//      "Wildcard types do not have a name");
```

Comment:
I assume these commented out INVARIANT's are intended to be re-instated once the generics support improves - if so, is it worth marking them all with FIXME or TODO comments as well to make that clearer? @thk123 is there any common practice here to include a link to a Jira or PR for in the comment to indicate what the work is covered under?